scripts/build_script.sh

```diff
@@ -28,6 +28,10 @@
 export RELEASE_VERSION=${RELEASE_VERSION:-$VERSION}
 export KEYCHAIN_NAME=${KEYCHAIN_NAME:-"login.keychain"}
 
+# Put homebrew Python ahead of system Python
+# https://stackoverflow.com/a/48101303
+export PATH="/usr/local/opt/python@3.11.8/libexec/bin:$PATH"
```

Comment:
Is it possible to get that version from a environment variable? 
My question is to check if we can reduce the places we need to worry about changing the version when we update it

Reply:
I agree that it'd be better to use a dynamic python version instead of hardcoding it here. If it's not bothering the logic maybe you could move those lines in the `scripts/builder_setup.sh` and use the `PYTHON_VERSION` env var defined there, wdyt?

Reply:
I'll merge this to unblock the python update work (since the feedback look is slow) and work on an improvement on a follow up PR.